src/views/Item/Item.js

```diff
@@ -29,17 +30,22 @@ function Item(props) {
     )
   }
 
+  let contentShorthandValue = content
+  if (
+    contentShorthandValue === undefined &&
+    [description, extra, header, meta].some((prop) => !_.isNil(prop))
+  ) {
+    contentShorthandValue = {}
```

Comment:
Only setting this when `content` is strictly `undefined` here and letting `ItemContent.create` handle everything else, because it allows the user to still use `null` (or a boolean) to prevent rendering when one of the other shorthands is set. Maybe `true` should also be included though?